src/js/components/TextInput/TextInput.js

```diff
@@ -129,6 +132,25 @@ const TextInput = forwardRef(
     );
 
     const [suggestionsAtClose, setSuggestionsAtClose] = useState();
+    const [tipContent, setTipContent] = useState(
+      format({ id: 'input.readOnlyCopyValidation', messages }),
```

Comment:
could we create variables for the formatted messages once and just reference those later in the code?

```
const readOnlyCopyValidation = format({ id: 'input.readOnlyCopyValidation', messages });
const readOnlyCopyPrompt =  format({ id: 'input.readOnlyCopy', messages });

Reply:
Yes, done